kubernetes/coredns.yaml.sed

```diff
@@ -51,6 +51,7 @@ data:
         log
         health
         kubernetes CLUSTER_DOMAIN SERVICE_CIDR POD_CIDR
+        prometheus :9153
```

Comment:
I added it because the Service below exports this port and I want to scrap these metrics.

I'm not sure if I have to add a parameter to bind to all interfaces. By default it binds to localhost only but I want to get this metrics via the exposed Service. I've noticed that kubeadm doesn't have it (https://github.com/kubernetes/kubernetes/blob/master/cluster/addons/dns/coredns.yaml.in#L63).